### ISSUE TITLE
openssl 1.1 problem

### ISSUE THREAD

**Issue by andrejfilipcic** [apel/ssm#96]
With openssl 1.1, the crypto.py complains on syntax error in this line:
https://github.com/apel/ssm/blob/055aeb2e8e90e81605ec7c6b6e78d7f66069118f/ssm/crypto.py#L221
```
# openssl x509 -checkend -noout -in /etc/grid-security/hostcert.pem
x509: Can't parse "-noout" as a number
x509: Invalid number "-noout" for -checkend
x509: Use -help for summary.
```

adding argument to checkend, eg "1" works as expected:
```
# openssl x509 -checkend 1 -noout -in /etc/grid-security/hostcert.pem
Certificate will not expire
```

If the line is replaced by 
`args = ['openssl', 'x509', '-checkend', '1', '-noout', '-in', certpath]`
it works for both openssl 1.0 and openssl 1.1

Could be this fixed?

**Comment by tofu-rocketry** [comment]
Ah, right, that looks like bit of a bug. I take it openssl 1.0 just assumed you meant 0 or 1?

I think that checking if the cert expires within the next hour (or day?) would be good considering the time it could take a message to traverse the message brokers.

**Comment by andrejfilipcic** [comment]
Yes, openssl 1.0.* and openssl 1.1.* Interval can be one day, sure.

**Comment by tofu-rocketry** [comment]
Resolved by #97.
